### PR TITLE
Update triplea_maps.yaml

### PR DESCRIPTION
**Pull request by KaiMAD**
I fixed the error in Update triplea_maps.yaml #552; also I put the URL first.

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/560)

### CODE REVIEW

File: triplea_maps.yaml [hunk 1]
```diff
@@ -327,6 +327,29 @@
     <br>Feedback highly welcome, if played.
     <br>Leave feedback at the forums: http://triplea.sourceforge.net/mywiki/Community
     <br>or at the Depot: http://sites.google.com/site/tripleaerniebommel/
++  url: https://github.com/KaiMAD/big_world_1939/blob/master/big_world_1939.zip+
```
Comment (DanVanAtta) [inline triplea_maps.yaml]: Still not correct :( 
There's an extra "plus" on each line.
This new block should look like all the other ones.. 
Also, the "-"  in front of the "url:" is missing.

Comment (KaiMAD) [inline triplea_maps.yaml]: Ah! What tedium. I will get this fixed soon.

**Comment by DanVanAtta** [comment]
#561 replaces, closing.

File: triplea_maps.yaml (hunk 1 above)
Comment (DanVanAtta) [inline triplea_maps.yaml]: yeah, sorry about the trouble, it's supposed to be an easy process. thanks @KaiMAD

**Pull request closed without merging**